12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,25 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
-
+echo [a-z]*
   2. All files whose name starts with an uppercase letter
   Answer:
-
+echo [A-Z]*
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
-
+echo *[aeiou]
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
-
+echo *[!aeiou]
   5. All files with extension ".c" that are in a folder named "src"
   Answer:
-
+echo src/*.c
   6. All files with extension ".c" under any folder
   Answer:
-
+echo */*.c
```

Comment:
that's correct! :) also note that there is a "hidden" [recursive glob](https://github.com/tocodeil/unix1-course-examples/issues/68) if you're into that kind of things :P